src/function/loadEmojione.js

```diff
@@ -33,49 +41,90 @@ function(emojione, uniRegexp, emojioneVersion, readyCallbacks, emojioneSupportMo
                 case '2.1.2': return 3;
                 case '2.1.3':
                 case '2.1.4':
-                case '2.2.7':
+                case '2.2.7': return 4;
+                case '3.1.0': return 5;
                 default: return 4;
             }
         }
-        options = getOptions(options);
 
+        function loadEmojione3Lib() {
```

Comment:
Duplicate code

Reply:
Do you suggest that it only loads version 3 if does not have emojione?

Reply:
I suggest to use one code for loading emojione, and only change cdn base and assets depending of emojione support mode